chromadb/test/property/test_embeddings.py

```diff
@@ -372,23 +391,30 @@ def wait_for_compaction(self) -> None:
 
     @rule(
         target=embedding_ids,
-        record_set=strategies.recordsets(collection_st),
+        record_set=strategies.recordsets(collection_st, can_ids_be_empty=True),
     )
     def add_embeddings(self, record_set: strategies.RecordSet) -> MultipleResults[ID]:
         res = super().add_embeddings(record_set)
         normalized_record_set: strategies.NormalizedRecordSet = invariants.wrap_all(
             record_set
         )
+
+        n_records = invariants.get_n_items_from_record_set(normalized_record_set)
+        ids = [id for id in res]
+
         print(
             "[test_embeddings][add] Non Intersection ids ",
-            normalized_record_set["ids"],
+            ids,
             " len ",
-            len(normalized_record_set["ids"]),
+            len(ids),
         )
-        self.log_operation_count += len(normalized_record_set["ids"])
-        for id in normalized_record_set["ids"]:
+
+        self.log_operation_count += n_records
+
+        for id in res:
```

Comment:
Need to make sure this is correct. @sanketkedia - not sure that this logic is doing what we expect.